### DISCUSSION TITLE
The problem in OSB estimation

### DISCUSSION THREAD

**Discussion by k9891** [groops-devs/groops#223] [Q&A]
Hello:
I am currently using GROOPS to estimate OSB products, but I have noticed a problem. 
If I implement pseudorange OSB for CAS and only estimate phase OSB, the final result is very poor.\
I think there is a problem with my strategy. I have turned off the satellites-end in Codebias and Tecbias parameters, and the other parameters are basically the same as the example.
Thanks for the Help

regards,
K9891

![微信截图_20231119100531](https://github.com/groops-devs/groops/assets/58100188/1c809fbb-5635-475d-90ee-dd533af53179)
<img width="359" alt="微信图片_20231119100601" src="https://github.com/groops-devs/groops/assets/58100188/efe8952c-68e5-49bf-ad5a-43ee57332388">

**Comment by zhedumi** [comment]
Hi,

Sorry but I need a bit more info regarding what you are trying todo. Are you computing PPP or a staiton network estimation? I presume you are using PPP since transmitter biases have been deactivated?
What products are you using for the transmitter biases and are you using the respective clock products with it? 
Also yozu might run into the problem of having too few observations to estimate them in the model in the same way as it was in this  [discussion](https://github.com/groops-devs/groops/discussions/190). Especially this [tip](https://github.com/groops-devs/groops/discussions/190#discussioncomment-6909426). Or can you show the log seciton which has
setup parameters in it?


Best regards,
Patrick
